### DISCUSSION TITLE
Project a marker that is outside the viewport

### DISCUSSION THREAD

**Discussion by belukha** [rnmapbox/maps#3693] [Q&A]
In the documentation, I found only the `getPointInView` function, which projects `[lon, lat]` coordinates that are within the viewport. If the coordinates are outside the viewport, it returns `[-1, -1]`. Is there a way to project a marker even if it is outside the viewport?

**Comment by Elter71** [comment]
Hi @belukha 
unfortunately, Mapbox does not provide a direct way to project coordinates outside the current map viewport.

**Reply by belukha** [reply]
@Elter71  
I managed to obtain coordinate projections that are outside the viewport on ios using this code:
```swift
let screenPosition = Projection.project(coordinate, zoomScale: 2.0)
```

docs: https://docs.mapbox.com/ios/maps/api/11.5.1/documentation/mapboxmaps/projection/project(_:zoomscale:)/

How could I implement the same using react native? 
btw, mapbox-gl gives control to get screen projection that are outside the viewport as well
